src/seedu/addressbook/commands/Command.java

```diff
@@ -39,12 +39,12 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
+//    public CommandResult execute(){
+//        throw new UnsupportedOperationException("This method should be implement in child classes");
+//    }
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.